service/storage_proxy.cc

```diff
@@ -3784,12 +3784,16 @@ class abstract_read_executor : public enable_shared_from_this<abstract_read_exec
         make_requests(digest_resolver, timeout);
 
         // Waited on indirectly.
-        (void)digest_resolver->has_cl().then_wrapped([exec, digest_resolver, timeout] (future<digest_read_result> f) mutable {
+        (void)digest_resolver->has_cl().then_wrapped([exec, digest_resolver, timeout] (future<result<digest_read_result>> f) mutable {
             bool background_repair_check = false;
-            try {
+            (void)utils::result_try([&] () -> result<> {
```

Comment:
(void) comment

Reply:
Here as well. Is there an advantage to use `result_try`?

Reply:
I tried to get rid of `result_try` here, but the problem is that there is more code to execute after the try..catch block. After I handle the failed result, I need to jump right after the try..catch and continue. To do this, I could use a goto or an immediately invoked lambda - however I'm not sure if those options are really better than `result_try`, so if there are no objections I'll keep the `result_try` as it is here now.

Reply:
Added a (void) comment, too.